GitUI/Editor/FileViewerInternal.cs

```diff
@@ -112,6 +112,13 @@ public void SetText(string text, Action openWithDifftool, bool isDiff = false)
             _currentViewPositionCache.Capture();
 
             OpenWithDifftool = openWithDifftool;
+            if (!AppSettings.ShowLineNumbers)
+            {
+                isDiff = false;
+            }
+
+            TextEditor.ShowLineNumbers = false;
```

Comment:
As intended?

Reply:
Yes, this hides an extra 3rd row (after the 2 own with diff) with numbers going 1,2,3..